app/components/agenda/agenda-overview/agenda-overview-item.js

```diff
@@ -81,6 +86,18 @@ export default class AgendaOverviewItem extends AgendaSidebarItem {
     this.agendaitemDocuments = sortedPieces;
   }
 
+  @task
+  *loadAgendaReleaseAndPublishedStatus() {
+    const meeting = yield this.args.currentAgenda?.createdFor;
+    if (meeting) {
+      const newsletter = yield meeting.newsletter;
+      const mailCampaign = yield meeting.mailCampagin;
+
+      this.agendaIsReleased = isPresent(meeting.releasedDecisions || meeting.releasedDocuments);
```

Comment:
I didn't really check. Does the one imply the other? I.e., `meeting.releasedDecisions` has to be true for `meeting.releasedDocuments` to be true?

I wrote it like this because I was trying to follow the specification in the [feature passport (subtitle Toepassing vertrouwelijkheidsniveaus)](https://smoothsailing.notion.site/Zichtbaarheid-Documenten-87528ca2443b4a848f925d04b613e357)